File: lib_common/src/main/java/com/fly/tour/common/mvp/INoDataView.java
package com.fly.tour.common.mvp;

import android.support.annotation.DrawableRes;

/**
 * Description: <INoDataView><br>
 * Author:      mxdl<br>
 * Date:        2018/3/11<br>
 * Version:     V1.0.0<br>
 * Update:     <br>
 */
public interface INoDataView {
    //显示无数据View
    void showNoDataView();
    //隐藏无数据View
    void hideNoDataView();

    //显示指定资源的无数据View
    void showNoDataView(@DrawableRes int resid);
}

File: lib_common/src/main/java/com/fly/tour/common/mvp/INoDataView.kt
package com.fly.tour.common.mvp

import androidx.annotation.DrawableRes

/**
 * Description: <INoDataView><br>
 * Author:      mxdl<br>
 * Date:        2018/3/11<br>
 * Version:     V1.0.0<br>
 * Update:     <br>
</INoDataView> */
interface INoDataView {
    //显示无数据View
    fun showNoDataView()

    //隐藏无数据View
    fun hideNoDataView()

    //显示指定资源的无数据View
    fun showNoDataView(@DrawableRes resid: Int)
}
